runtime/common/src/impls.rs

```diff
@@ -44,8 +44,11 @@ where
 {
 	fn on_unbalanceds<B>(mut fees_then_tips: impl Iterator<Item = NegativeImbalance<R>>) {
 		if let Some(fees) = fees_then_tips.next() {
-			let trasury_amount = super::TREASURY_FEE_RATIO * fees.peek();
-			let mut split = fees.split(trasury_amount);
+			// for fees, split the destination
+			let mut split = fees.ration(
+				super::TREASURY_FEE_RATIO.deconstruct(),
```

Comment:
can we import `TREASURY_FEE_RATIO` above and avoid referencing it twice with `super::`?

Reply:
Also, I think we could actually make this a const/function that we can then apply - as a sort of bimap - against a fee value. Doing that would also dry the duplicated logic we now have in the tests.

Reply:
Ah wait, the `.ration(` function is the bimap I was hoping to have. It could still be helpful to dry this into a function and thus avoid the duplication in the tests but leave it up to you.

Reply:
Regarding the first point, I agree. In fact, I would avoid also including the `super::*` and import explicitly each constant used in this module, in order to identify better where they come from. Anyway, I do not know if that `super::*` for common types/constants is something used (recommended) by the framework.

Reply:
Regarding the second point, I do not understand exactly where the duplication is. Anyway, If I use the same "logic" function in both places, code and test, I would never notice a possible error if somebody in the future changes that logic, since the test will still pass.

Reply:
Regarding the second point, and for the current code size. I think it could more messy if changed because in the non-test code you should need to call the extracted function and later transform each value with `.deconstruct()`, and the result seems less readable than what we're trying to archive.

Reply:
Oke, sounds fair 👍